makefiles/checks.mk

```diff
@@ -150,6 +138,29 @@ check_node:
 	fi; \
 	}
 
+.PHONY: check_path_up
+# Internal helper: Checks if PATH is running at localhost:3069
+check_path_up:
+	@if ! nc -z localhost 3069 2>/dev/null; then \
+		echo "########################################################################"; \
+		echo "ERROR: PATH is not running on port 3069"; \
+		echo "Please make sure localnet_config.yaml contains at least 1 path gateway and start localnet with:"; \
+		echo "  make localnet_up"; \
+		echo "########################################################################"; \
+		exit 1; \
+	fi
+
+.PHONY: check_relay_util
+# Internal helper: Checks if relay-util is installed locally
+check_relay_util:
+	@if ! command -v relay-util &> /dev/null; then \
+		echo "####################################################################################################"; \
+		echo "Relay Util is not installed." \
+		echo "To use any Relay Util make targets to send load testing requests please install Relay Util with:"; \
+		echo "go install github.com/commoddity/relay-util/v2@latest"; \
```

Comment:
Wdyt about adding this to [`install_ci_deps`](https://github.com/pokt-network/poktroll/blob/main/Makefile#L82)?

Reply:
Will do so when it becomes part of CI.

For now, this isn't even documented - just getting in a baseline "IYKYK" sort of thing.